.github/workflows/smoke-test.yml

```diff
@@ -21,4 +21,3 @@ jobs:
       - uses: AriPerkkio/eslint-remote-tester-run-action@v3
         with:
           issue-title: 'Results of weekly scheduled smoke test'
-          eslint-remote-tester-config: smoke-test/eslint-remote-tester.config.js
```

Comment:
The action doesn't actually pick these `.ts` configurations by default since action parameter has default value for `.js` config: https://github.com/AriPerkkio/eslint-remote-tester-run-action/blob/e72eaaa36788a76ef9044271bebbbd83377e7d6d/action.yml#L14-L17

So this needs to be specified here: 
```yml
    eslint-remote-tester-config: smoke-test/eslint-remote-tester.config.ts
```

Reply:
Ah yeah I wondered if that might be the case 👍